deployments/systemd/README.md

```diff
@@ -85,6 +85,19 @@ nvidia-mig-parted apply -f /etc/nvidia-mig-manager/config.yaml -k /etc/nvidia-mi
 ```
 (An alternative is to ensure that the required environment variables are passed to `sudo` by using `sudo -E` instead)
 
+**NOTE:** When running the nvidia-mig-parted, if a message appears that is similar to
```

Comment:
Question: Do we want to add a `Troubleshooting` section to the readme to collect this type of information?

Reply:
I am fine with a Trobuleshooting section